docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -546,7 +1048,7 @@ yb-voyager import schema --export-dir <EXPORT_DIR> \
         --post-snapshot-import true
```

Comment:
Can we please remove this Important note from all the live migration workflows in the `Prepare the target database` step - 
```
{{<note title="Important">}}
Add the following flags to the cluster before starting migration, and revert them after the migration is complete.
```